src/python/WMCore/Storage/StageOutMgr.py

```diff
@@ -188,7 +190,7 @@ def __call__(self, fileToStage):
         Use call to invoke transfers
 
         """
-        lastException = ""
+        lastException = None
```

Comment:
Is this change causing the following error to be reported in the pylint section:
```
 E0702, line 253 in StageOutMgr.__call__: Raising NoneType while only classes or instances are allowed 
```
?
If so, I'd then suggest to switch it back to empty string. AFAICT there is no difference

Reply:
I see. There was already a pylint report on that same line.
In this case, feel free to leave the original code `""` or set it to the standard exception that you suggest. That default value will never be raised anyways :)

Reply:
Perfect, thank you!